allennlp_models/rc/predictors/transformer_qa.py

```diff
@@ -47,23 +53,51 @@ def predict_json(self, inputs: JsonDict) -> JsonDict:
         assert len(results) == 1
         return results[0]
 
+    @overrides
+    def predictions_to_labeled_instances(
+        self, instance: Instance, outputs: Dict[str, numpy.ndarray]
+    ) -> List[Instance]:
+        new_instance = instance.duplicate()
+        span_start = int(outputs["best_span"][0])
+        span_end = int(outputs["best_span"][1])
+
+        start_of_context = (
+            len(self._dataset_reader._tokenizer.sequence_pair_start_tokens)
+            + len(instance["metadata"]["question_tokens"])
+            + len(self._dataset_reader._tokenizer.sequence_pair_mid_tokens)
+        )
+
+        answer_span = SpanField(
+            start_of_context + span_start,
+            start_of_context + span_end,
+            instance["question_with_context"],
+        )
+        new_instance.add_field("answer_span", answer_span)
+        return [new_instance]
+
     @overrides
     def _json_to_instance(self, json_dict: JsonDict) -> Instance:
-        raise NotImplementedError(
-            "This predictor maps a question to multiple instances. "
-            "Please use _json_to_instances instead."
+        logger.warning(
```

Comment:
@dirkgr I'm not sure if just adding a warning instead of an error actually breaks anything.

Reply:
I don't think it breaks anything, but people don't read warnings.